gcloud-java-storage/src/main/java/com/google/gcloud/storage/BlobInfo.java

```diff
@@ -589,6 +601,18 @@ public Long updateTime() {
   }
 
   /**
+   * Returns {@code true} if the current blob represents a directory. This can only happen if the
+   * blob is returned by {@link Storage#list(String, Storage.BlobListOption...)} when the
+   * {@link Storage.BlobListOption#currentDirectory()} option is used. If {@code true} only
```

Comment:
I would use `{@code true}` only when it is in a code context (e.g. boolean param, return type,...) but not when "true" is part of the sentence (and could be replaced with e.g. "this is the case").